src/App/App.js

```diff
@@ -1,31 +1,51 @@
 import React from 'react'
 import { Route, Switch, Link, Redirect } from 'react-router-dom'
 import './App.css'
+import Profile from '../Profile/Profile'
 
 import Gallery from '../Gallery/Gallery'
 
 import userData from '../userData'
 
 class App extends React.Component {
   state = {
+    currentUser: {},
     users: []
   }
 
   componentDidMount() {
-    this.setState({ users: userData })
+    // fetch
+    this.setState({ currentUser: userData[1], users: userData })
   }
 
-  render() {
-    const { users } = this.state
+  // addPlaydateForCurrentUser = (newPlaydate) => {
```

Comment:
Foreseeing a potential need to add currentUser appointments somewhere in state, probably in App